spec/Powertrain/FuelSystem.vspec

```diff
@@ -76,8 +76,12 @@ ConsumptionSinceStart:
   type: sensor
   unit: l
   description: Fuel amount in liters consumed since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
```

Comment:
Is it sufficient to have consumption since start of current trip, or do we also need a signal for "consumption since trip meter reset". Similar for the signal `AverageConsumption` above. Today we state that period is a variation point, shall we better specify that it is for current trip or split it to two signals, one for "average consumption for current trip" and one for "average consumption since trip meter reset"